code/services/QueuedJobService.php

```diff
@@ -315,6 +321,57 @@ public function checkJobHealth($queue = null) {
 	}
 
 	/**
+	 * Checks through all the scheduled jobs that are expected to exist
+	 */
+	public function checkdefaultJobs($queue = null) {
+		$queue = $queue ?: QueuedJob::QUEUED;
+		if (count($this->defaultJobs)) {
+
+			$activeJobs = QueuedJobDescriptor::get()->filter(array(
+					'JobStatus' => array(
+					QueuedJob::STATUS_NEW,
+					QueuedJob::STATUS_INIT,
+					QueuedJob::STATUS_RUN,
+					QueuedJob::STATUS_WAIT,
+				),
+				'JobType' => $queue
+			));
+
+			foreach ($this->defaultJobs as $title => $jobConfig) {
+				if(count(array_intersect(array('filter', 'type'), array_keys($jobConfig))) !== 2) {
```

Comment:
Is... this just a long winded way of saying `if (!isset($jobConfig['filter']) || !isset($jobConfig['type']))` ?